backend/src/notify/telegram.ts

```diff
@@ -0,0 +1,29 @@
+import { NotifierInterface, Credentials } from './interface'
+import * as fetch from 'node-fetch'
+
+export default class TelegramNotifier implements NotifierInterface {
+  private credentials: Credentials
+  private chat_id: Promise<number>
+
+  constructor (credentials: Credentials) {
+    this.credentials = credentials
+    this.chat_id = this.getChatId(credentials)
+  }
+
+  async send (message: string) {
+    const endpoint = `https://api.telegram.org/bot${this.credentials.username}:${this.credentials.password}/sendMessage?chat_id=${await this.chat_id}&text=${message}`
+    try {
+      await fetch(endpoint)
+    } catch {
+      return false
```

Comment:
maybe add a console log to know when it fail to send the message ?

Reply:
yes good idea 👍🏻